flytekit/core/local_cache.py

```diff
@@ -0,0 +1,29 @@
+from typing import Callable, List, Optional
+
+from joblib import Memory
+
+# Location in the file system where serialized objects will be stored
+# TODO: read from config
+CACHE_LOCATION = "~/.flyte/local-cache"
+
+
+class LocalCache(object):
+    _memory: Memory
+    _initialized: bool = False
+
+    @staticmethod
+    def initialize():
+        LocalCache._memory = Memory(CACHE_LOCATION, verbose=5)
```

Comment:
should we make the verbosity configurable too?

Reply:
I don't think we gain a lot from making this configurable, especially given that configuration files are not used at all in local executions. I'm going to leave a TODO as well.